src/go/configmanager/flags/flags.go

```diff
@@ -97,6 +97,9 @@ var (
 
 	EnableGrpcForHttp1 = flag.Bool("enable_grpc_for_http1", true, `Enable gRPC when the downstream is HTTP/1.1. The default is on.`)
 
+	ConnectionBufferLimitBytes = flag.Int("connection_buffer_limit_bytes", 1*1024*1024, `Configure the maximum amount of data that is buffered for each request/response body. 
+			This should be increased for large bodies used by gRPC transcoding. Default is 1 MB.`)
```

Comment:
I am debating: if this flag is not set,  should we introduce espv2 default, or just use envoy default. 

IMO, we (espv2) is not in the business of deciding such default.  we should let envoy to decide it.

Reply:
Well Envoy [defaults to 1 MB](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/listener/v3/listener.proto). We should document this for our users, right? Otherwise they don't know what the default will be.

Reply:
Sorry,  I did not state it clearly.  I would prefer

If this flag is not set, ESPv2 don't set this setting in envoy config.  it will use envoy default value.
We don't want to define our own default value.  which may be different with future envoy default value.

Reply:
I think we should explicitly set a default. What if Envoy lowers it in the future? It could break our clients.

I doubt Envoy would change it anyways, so I don't think we need to worry. It's easier to unit test if we always include it.

Reply:
If envoy ever change its default, it must be a reason, to fix somethings. we may want to benefits from that changes too. For backward compatibility,  I don't think Envoy will change a default to break anybody.  that change will not be a breaking change for any users, including ours. 
For your unit-test argument, I don't think we have unit-test to test 1MB default. with actual data flow.  You may mean the unit-test verifying envoy config setting.  "not setting it" can also be verified in unit-test too.

Reply:
Makes sense, done.